lib/errors.js

```diff
@@ -148,31 +148,40 @@ Object.defineProperties(DOMException.prototype, {
 	},
 });
 
-DOMException.INDEX_SIZE_ERR = 1;
-DOMException.DOMSTRING_SIZE_ERR = 2;
-DOMException.HIERARCHY_REQUEST_ERR = 3;
-DOMException.WRONG_DOCUMENT_ERR = 4;
-DOMException.INVALID_CHARACTER_ERR = 5;
-DOMException.NO_DATA_ALLOWED_ERR = 6;
-DOMException.NO_MODIFICATION_ALLOWED_ERR = 7;
-DOMException.NOT_FOUND_ERR = 8;
-DOMException.NOT_SUPPORTED_ERR = 9;
-DOMException.INUSE_ATTRIBUTE_ERR = 10;
```

Comment:
This value is used in some places, I even spotted one in the diff of one of the tests. How will this work?

Update: I just spotted the thing you do using `Object.entries` below, clever move. I'm curious to see if this works as expected...

Multiple contributors, including me, have been tempted to do something about this in the past.
Not sure if it is good to do it in this PR...

Reply:
Is there a reason this should not work? 😄
From my tests it seems to work just as expected, but you can verify that, too - just to be sure.